packages/airnode-deployer/src/handlers/gcp/index.ts

```diff
@@ -52,3 +52,27 @@ export async function processProviderRequests(req: Request, res: Response) {
   const body = { ok: true, data: providers.scrub(updatedState) };
   res.status(200).send(body);
 }
+
+export async function testApi(req: Request, res: Response) {
+  const apiKey = req.header('x-api-key');
+  if (!apiKey || apiKey !== config.getEnvValue('HTTP_GATEWAY_API_KEY')) {
```

Comment:
This is here because we can't whitelist by this header in the gateway, right? (in aws this is done by aws)

Reply:
It would deserve a comment.

Reply:
Yes, that's why we have to do it manually here. I'll add a comment.